tests/test_dom0_systemd.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+import subprocess
+import unittest
+
+# .spec file shows which units are shipped with rpm
+SYSTEM_UNITS = ["securedrop-logind-override-disable.service"]
+
+USER_UNITS = [
+    "securedrop-user-xfce-icon-size.service",
+    "securedrop-user-xfce-settings.service",
+    "sdw-notify.timer",
+]
+
+# list all system unit files (append "--user" for user files)
+# that match our naming convention
+SYSTEMD_LIST_UNITS = ["systemctl", "--legend=false", "list-unit-files", "--all", "*securedrop*"]
```

Comment:
This approach is great too, I wonder if we should be renaming the `sdw-notify.timer` unit as out of compliance with this then.